src/AutoRest.CSharp/Mgmt/Output/Models/MgmtPropertyBag.cs

```diff
@@ -43,8 +44,8 @@ protected override TypeProvider EnsurePackModel()
             foreach (var parameter in _paramsToKeep)
             {
                 var inputParameter = _operation.Parameters.FirstOrDefault(p => string.Equals(p.Name, parameter.Name, StringComparison.OrdinalIgnoreCase));
-                var description = parameter.Description ?? $"The {parameter.Name}";
-                var property = new InputModelProperty(parameter.Name, parameter.Name, description.ToString(), inputParameter!.Type, parameter.DefaultValue == null, false, false)
+                var description = inputParameter is not null && !string.IsNullOrEmpty(inputParameter.Description) && parameter.Description is not null ? parameter.Description.ToString() : $"The {parameter.Name}";
```

Comment:
I think `inputParameter` is not null here, otherwise code gen wil fail on next line, e.g. `inputParameter!.Type`
```suggestion
                var description = !string.IsNullOrEmpty(inputParameter!.Description) && parameter.Description is not null ? parameter.Description.ToString() : $"The {parameter.Name}";
```

Reply:
Fixed